core/src/test/java/org/apache/calcite/plan/RelWriterTest.java

```diff
@@ -429,6 +435,11 @@
     return Stream.of(SqlExplainFormat.TEXT, SqlExplainFormat.DOT);
   }
 
+  private static RexNode apply(Map<String, Object> map, RexBuilder rexBuilder, List<RelNode> inputs) {
```

Comment:
need better name than `apply`. Also a bit of javadoc describing intend. In the javadoc I'd mention that it is intended to implement the translator interface.